### ISSUE TITLE
Background Scanner not working

### ISSUE THREAD

**Issue by michag86** [nextcloud/files_antivirus#12]
### Steps to reproduce
1. Upload a file
2. Configure files_antivirus and clamav
3. check nextcloud.log and files_antivirus table

### Expected behaviour
Files will be scanned and oc_files_antivirus is filled.

### Actual behaviour
Files are not scanned and an error is logged.

### Server configuration
**Operating system**: CentOS 7.2
**Web server:** Apache
**Database:** MariaDB
**PHP version:** 5.4
**Nextcloud version:** 10.0.1
**Updated from an older Nextcloud/ownCloud or fresh install:** Migration from ownCloud
**Where did you install Nextcloud from:** zip from nextcloud.com
**Signing status:** No errors have been found.

Tested with files_antivirus stable10 and master.

#### Nextcloud log (data/nextcloud.log)
```
{"reqId":"IffqSbwSKP1Cw17UpXd4","remoteAddr":"","app":"files_antivirus","message":"OCA\\Files_Antivirus\\BackgroundScanner::run, exception: File with id \"591\" has not been found.","level":3,"time":"2016-11-28T12:15:12+01:00","method":"--","url":"--","user":"--"}
```
It's always the same id and the oc_files_antivirus table is empty.

I think this can resolve the problem:
https://github.com/owncloud/files_antivirus/pull/118

**Comment by tx7** [comment]
I having the same issue on NC11.

Debian Jessie
Nginx
MySQL 5.5

`Error	files_antivirus	OCA\Files_Antivirus\BackgroundScanner::run, exception: File with id "6" has not been found`

**Comment by tx7** [comment]
Hi,

Any solution?

**Comment by SomeStrangeName** [comment]
Maybe this helps: https://github.com/nextcloud/files_antivirus/issues/13

**Comment by tx7** [comment]
Still getting errors on NC11.0.1

`OCA\Files_Antivirus\BackgroundScanner::run, exception: File with id "4" has not been found.`

**Comment by SomeStrangeName** [comment]
Also with 11.0.1 and 11.0.2 of nextcloud I see this message in the logs. Initially it looked like rescanning files with `sudo -u www-data php console.php files:scan --all` seems to fix it but at the end nope :-/

**Comment by tx7** [comment]
The antivirus has been fixed in 11.0.2.

**Comment by rullzer** [comment]
Should be fixed
